Assets/Events/Flavour.json

```diff
@@ -37,14 +37,14 @@
   {
     "name": "magic-dust",
     "headline": "Magic Dust Bust Plus",
-    "article": "Big victory today for our boys in bronze as the Dust Enforcement Administration bags a massive load from Magic Dust traffickers caught attempting to smuggle 7000 gold pieces of product inside a bag of holding. The traffickers state that they did not pack their bags themselves.",
+    "article": "Big victory today for our boys in bronze as the Dust Enforcement Administration bags a massive load from Magic Dust traffickers caught attempting to smuggle 7000 gold pieces of product inside a <b>bag of holding</b>. The traffickers state that they did not pack their bags themselves.",
```

Comment:
I usually reserve the bolding of keywords to when they are used to signify chain events or significant things, not sure it should be used here

Reply:
Was just trying to accentuate the joke since I've realised it's hard to spot. I've gone and italicised instead.